protocol/app/ante.go

```diff
@@ -27,11 +28,12 @@ import (
 // struct embedding to include the normal cosmos-sdk `HandlerOptions`.
 type HandlerOptions struct {
 	ante.HandlerOptions
-	Codec            codec.Codec
-	AuthStoreKey     storetypes.StoreKey
-	ClobKeeper       clobtypes.ClobKeeper
-	PerpetualsKeeper perpetualstypes.PerpetualsKeeper
-	PricesKeeper     pricestypes.PricesKeeper
+	Codec             codec.Codec
+	AuthStoreKey      storetypes.StoreKey
+	AccountplusKeeper *accountpluskeeper.Keeper
```

Comment:
Are there any functional benefits to creating interface, such as anticipated additional implementations?

I'm hesitant to create an interface purely to provide a typing workaround, especially if the interface is not functioning as an interface. This could just be my Java dogma though.

Reply:
One functional benefit being that it's easy to create a mock instance. For this reason we have a keeper interface for almost all keeper objects [here](https://github.com/dydxprotocol/v4-chain/blob/f162a117a5d44dd6db9e9651ff77597f128990ea/protocol/mocks/Makefile#L41). 

The `ante_test.go` doesn't use mocks so it's not immediately helpful here. Overall, slight preference to define interface (since pretty easy) for style consistency, but up to you!

Reply:
Gotcha.

A little short on time esp with being still kinda new to Go. I will leave a todo and address in the future.